temporalcli/commands_test.go

```diff
@@ -216,6 +216,9 @@ func (s *SharedServerSuite) SetupSuite() {
 				// Allow a high rate of change to namespaces, particularly
 				// for the task-queue command tests.
 				"frontend.namespaceRPS.visibility": 10000,
+				// this is overridden since we don't want caching to be enabled while testing DescribeTaskQueue
+				// behaviour related to versioning
+				"matching.TaskQueueInfoByBuildIdTTL": 0 * time.Second,
```

Comment:
So we disable search attribute caching for all users of the dev server, not just our test suite. I wonder if we should do that here too. Though it may give the dev server users unreasonable expectations about eventual consistency for task queue info.

Reply:
I think it's reasonable for the dev server to have the caching for now -- can always change it later, but I think this caching mechanism may change in the future as well